app/models/hacker_news_repo.rb

```diff
@@ -0,0 +1,16 @@
+class HackerNewsRepo
+  def initialize(hacker_news_uri = 'https://hacker-news.firebaseio.com/v0', story_max = 30)
```

Comment:
Not sure I like those defaults in the parameter names. I would either put them in constants for now